docroot/modules/humsci/hs_views_helper/hs_views_helper.module

```diff
@@ -28,6 +31,38 @@ function hs_views_helper_help($route_name, RouteMatchInterface $route_match) {
   }
 }
 
+/**
+ * Implements hook_views_post_render().
+ *
+ * Views render arrays contain a cache tag "node_list". This cache tag is
+ * cleared every time ANY node is created, edited or deleted. When this happens
+ * every view on the site gets its cache flushed. This causes poor performance
+ * since a view would get flushed even if it has no relation to that node. To
+ * assist in cache tags, we create a custom cache tag based on the node type
+ * filter on the view. Its a small improvement but will have huge impact in
+ * keeping cached renders much longer.
+ *
+ * @see hs_views_helper_node_presave()
+ */
+function hs_views_helper_views_post_render(ViewExecutable $view, &$output, CachePluginBase $cache) {
+  if (isset($view->filter['type']) && $view->filter['type']->value) {
+    $node_list_position = array_search('node_list', $output['#cache']['tags']);
+    unset($output['#cache']['tags'][$node_list_position]);
+    foreach ($view->filter['type']->value as $node_type) {
```

Comment:
Does this need to check to see if the filter is on the entity type node before adding it to the cache tags?

Reply:
ahh probably. a filter on media type would not work as well.

Reply:
Should be good now.